### PR TITLE
feat: rtd-1993

### PR DESCRIPTION
**Pull request by mamari90**
### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

#### Has This Been Tested?

- [ ] Yes
- [ ] No

### Other information
Target: 
```

```

#### How Has This Been Tested?

- [ ] Unit Test Suite
- [ ] Integration Test Suite
- [ ] Api Tests
- [ ] Load Tests

[Link to test results](https://pagopa.atlassian.net/browse/RTD-XXXX)

---

### CODE REVIEW

**Comment by gammam** [comment]
Following istructions on [Confluence](https://pagopa.atlassian.net/wiki/spaces/IDPAY/pages/564822701/Portale+Enti+-+Informazioni+utili+di+accesso) ; The website is provided by the storage and it is available ( i logged in ).
@and-mora please check and review :)

File: src/domains/idpay-common/04_cdn.tf [hunk 1]
```diff
@@ -49,7 +49,7 @@ locals {
  */
 // public storage used to serve FE
 module "idpay_cdn" {
-  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//cdn?ref=v6.18.0"
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//cdn?ref=f9ea700"
```
Comment (and-mora) [inline src/domains/idpay-common/04_cdn.tf:52]: is it wanted the different ref at cdn module? one for sha and one for version?

**Comment by and-mora** [comment]
> Following istructions on [Confluence](https://pagopa.atlassian.net/wiki/spaces/IDPAY/pages/564822701/Portale+Enti+-+Informazioni+utili+di+accesso) ; The website is provided by the storage and it is available ( i logged in ). @and-mora please check and review :)

i confirm the website reachability in dev environment

File: src/domains/idpay-common/04_cdn.tf (hunk 1 above)
Comment (mamari90) [reply src/domains/idpay-common/04_cdn.tf:52]: it is temporary. i was waiting for the pr to be approved on the module, so that the new tag is available

**Comment by antonioT90** [comment]
This change breaks the APIm policy used when invoking the tokenExchange behavior, or the API invoked when accessing to IDPAY from SELFCARE.

This is due to the not more authorized access to this link:
https://cstardweuidpayidpaycdnsa.blob.core.windows.net/idpay-fe-oidc-config/openid-configuration.json

I cannot open the ticket RTD-1993, is this change intended to make private the access to the storage or explicitly to require an authenticated request towards the blob?

**Comment by mamari90** [comment]
@antonioT90 yes, the purpose of this pr is to make private access to the storage. As we discussed via chat, a possible solution is to use the cdn url instead of the direct storage account url.

we can put this pr on hold until that change is done or i can edit it so that only the cdn storage account will be public, and the others are updated to private

@gammam @and-mora what do you suggest?

**Comment by gammam** [comment]
@mamari90 i suggest to use cdn url and close this task. Meanwhile i have already registered IDP-1884 to move the file on a different storage.

**Comment by gammam** [comment]
@mamari90 we need to change file location in order to be on CDN , we will track this change in https://pagopa.atlassian.net/browse/IDP-1884. 
Next we can fix this PR

**Pull request merged**